docs/src/pages/[platform]/guides/css-in-js/index.page.mdx

```diff
@@ -2,6 +2,7 @@
 title: CSS in JS
 metaDescription: One of the goals of Amplify is to be a good citizen of the environment it exists in. As such, Amplify works hard to not interfere with other tools that are being used.
 supportedFrameworks: react|angular|vue
```

Comment:
After checking this page, should `angular` and `vue` be supported here? It looks like it's showing react imports